playground/frontend/lib/modules/sdk/models/sdk.dart

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+enum SDK {
+  java,
+  go,
+  python,
+  scio,
+}
+
+extension SDKToString on SDK {
```

Comment:
I personally find these unique dart syntax sugar features in the language.  However, I wonder if those coming from traditional object oriented languages as is typical with this Apache Beam repo may not be as familiar.

Reply:
I know that Scala and Kotlin have something like this. I think can refactor it to a regular function if needed.